lib/hooks/useKeyboardKey.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 
-const useEscapeKey = callback => {
+const useKeyboardKey = ({ callback, keyName, keyAbb, keyCode }) => {
```

Comment:
Nice refactor, it's a good idea to make this one more generic :+1: 

I assume that `keyAbb` stands for "Key abbreviation", but this might not be very clear for everyone. What about this:

We change the props to:
```es6
({ callback, keyMatcher })
```

keyMatcher should an object like:
```es6
{
  key: '',
  keyName: '',
  keyCode: '',
}
```

From this file, we also export:
```es6
export const ESCAPE_KEY_MATCHER = {
  keyName: 'Escape', 
  key: 'Esc', 
  keyCode: 27
}
```

Then all we have to do to detect the escape key is:
```es6
useKeyboardKey({ callback: () => {}, keyMatcher: ESCAPE_KEY_MATCHER })
```